### PR TITLE
Remove Requires Windows 11 Only text from Glass theme

### PR DESCRIPTION
**Pull request by heftymouse**
## Description

The text stating that the Glass theme requires Windows 11 has been removed.

## Motivation and Context

Glass theme is now available on Windows 10 as well thanks to Acrylic support being added.

### CODE REVIEW

File: src/routes/docs/configuring/custom-themes/+page.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Files has built-in support for custom fonts, to apply a custom font start by ope
 
 We put together some popular color combinations to help users create custom themes.
 
-#### **Glass (requires Windows 11)**
+#### **Glass**
```
Comment (yaira2) [inline src/routes/docs/configuring/custom-themes/+page.md:35]:
```suggestion
#### **Glass** (requires enabling the Acrylic backdrop on Windows 10)
```

Comment (TheOnlyTails) [reply src/routes/docs/configuring/custom-themes/+page.md:35]: Maybe this note should be farther down as not to clutter up the title?

Comment (yaira2) [reply src/routes/docs/configuring/custom-themes/+page.md:35]: @heftymouse

Comment (heftymouse) [reply src/routes/docs/configuring/custom-themes/+page.md:35]: That makes sense, it can be below the table

**Comment by yaira2** [comment]
@heftymouse I cover this in #505.

**Pull request closed without merging**